src/LibraryManager.Vsix/Errors/ErrorList.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.Web.LibraryManager.Contracts;
 using Microsoft.Web.LibraryManager.LibraryNaming;
 
-namespace Microsoft.Web.LibraryManager.Vsix
+namespace Microsoft.Web.LibraryManager.Vsix.Errors
 {
     internal class ErrorList
```

Comment:
I'm inclined to go with the `ErrorList` namespace to match the original folder name as this is specifically for integrating into the Error List feature in Visual Studio.  I would actually prefer to rename the class here, maybe something like `ErrorListPropagator` or `ErrorListManager`.

Reply:
Done using `ErrorListPropagator`.

A bit of a nightmare dealing with the changes that VS wants to make to the VSIX project file... removing blank lines, reformatting, and inserting a strange `&#xD;&#xA;` character.